spec/models/queen_spec.rb

```diff
@@ -4,4 +4,30 @@
   it 'has type of Queen' do
     expect(subject.type).to eq 'Queen'
   end
+
+  describe '#valid_move?' do
+    let(:queen) { FactoryGirl.create(:queen, position_x: 3, position_y: 3) }
+
+    it 'returns true for vertical moves' do
+      expect(queen.valid_move?(3, 6)).to eq true
+    end
+
+    it 'returns true for horizontal moves' do
+      expect(queen.valid_move?(4, 3)).to eq true
+    end
+
+    it 'returns true for diagonal moves' do
+      expect(queen.valid_move?(1, 5)).to eq true
+    end
+
+    it 'returns false for moves that are not straight lines' do
+      expect(queen.valid_move?(4, 5)).to eq false
+    end
+
+    it 'returns false if obstructed' do
+      FactoryGirl.create(:pawn, position_x: 3, position_y: 4, game: queen.game)
```

Comment:
Again, don't need and also don't need to override the game param during create. I don't really care which we do things, but we should standardize one way or the other :smile: